raiden_installer/__init__.py

```diff
@@ -24,6 +25,7 @@ class TokenSettings:
 
 @dataclass
 class Settings:
+    name: str  # name of the corresponding toml file
```

Comment:
It's not really the name of the file, but just the basename. The suffix is excluded, AFAICS.